api/survey.ts

```diff
@@ -0,0 +1,59 @@
+import handler from './handler';
+import db from './db';
+
+export default handler(async (request : any) => {
+
+  let connection = db();
+  const client = await connection.connect()
+
+  // Take data from request and build sql call from it
+  // TODO: Move this data structure to it's own class
+  const {
+    //Sharded Values
+    patientid,
+    completiondate,
```

Comment:
I think these values should also be typed.

Reply:
Agreed, planned to move the definition out to it's own class first thorugh